scripts/ansible/oe-windows-acc-setup.yml

```diff
@@ -36,8 +36,8 @@
     - name: OE setup | Run the install-windows-prereqs.ps1 script (this may take a while)
       script: ../install-windows-prereqs.ps1
         -InstallPath         "C:\openenclave"
-        -WithFLC             "{{ 1 if dcap_testing_node is defined and dcap_testing_node == true else 0 }}"
-        -WithAzureDCAPClient "{{ 1 if dcap_testing_node is defined and dcap_testing_node == true else 0 }}"
+        -LaunchConfiguration "{{ SGX1FLC if dcap_testing_node is defined and dcap_testing_node == true else SGX1 }}"
+        -DCAPClientType      "{{ Azure }}"
```

Comment:
This will install the Azure DCAP client on all Windows CI systems, which will be useful for upcoming host verify tests.